plugin/trino-databend/src/main/java/io/trino/plugin/databend/DatabendEngineType.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.databend;
+
+public enum DatabendEngineType
+{
+    FUSE("FUSE"),
+    MEMORY("Memory"),
+    RANDOM("Random"),
+    DELTA("Delta"),
+    ICEBERG("Iceberg");
```

Comment:
Do we have tests for all engine types?

Reply:
Only test FUSE engine here, maybe I should remove other engines?

Reply:
Yes.